components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/common/IdentityConnectorConfig.java

```diff
@@ -16,13 +16,17 @@
 
 package org.wso2.carbon.identity.governance.common;
 
+import org.wso2.carbon.identity.application.common.model.Property;
 import org.wso2.carbon.identity.core.ConnectorConfig;
 import org.wso2.carbon.identity.governance.IdentityGovernanceException;
 
 import java.util.List;
 import java.util.Map;
 import java.util.Properties;
 
+/**
+ * IdentityConnectorConfig.
```

Comment:
Add a meaningful class comment

Reply:
Fixed in commit 8032899333352cf8a310623b49856dfb99ec9767